vilebot/src/test/java/vilebot/handlers/user/NewsTest.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) 2019 Newterns
+ *
+ * This file may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package vilebot.handlers.user;
+
+import com.oldterns.vilebot.handlers.user.News;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pircbotx.Channel;
+import org.pircbotx.User;
+import org.pircbotx.hooks.events.MessageEvent;
+
+import static org.mockito.Mockito.*;
+
+public class NewsTest
```

Comment:
Maybe add additional test cases that the number of headlines when parsing the news feed is the expected number (since you can't know what the headlines will be)/expected format and that the correct error is reported in a negative test case